odoo/addons/base/models/ir_cron.py

```diff
@@ -530,3 +602,20 @@ class ir_cron_trigger(models.Model):
     @api.autovacuum
     def _gc_cron_triggers(self):
         self.search([('call_at', '<', datetime.now() + relativedelta(weeks=-1))]).unlink()
+
+
+class ir_cron_progress(models.Model):
+    _name = 'ir.cron.progress'
+    _description = 'Actions Progress'
+    _rec_name = 'cron_id'
+
+    cron_id = fields.Many2one("ir.cron", index=True)
+    remaining_tasks = fields.Integer()
+    progress = fields.Integer()
+
+    def recall(self):
+        return self.remaining_tasks and self.progress
```

Comment:
Why is this method public, and what's its purpose?  Docstring please.

Reply:
relics of a bigone era :grimacing:, removed